content/en/continuous_integration/pipelines/custom.md

```diff
@@ -90,7 +91,17 @@ To send pipeline events programmatically to Datadog, ensure that your [`DD_API_K
    EOF
    {{< /code-block >}}
 
-1. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+3. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+
+## Running pipelines
+Pipeline events sent with the `status` set to `running` have the same `unique_id` as the final pipeline event.
+Additionally, running pipelines can be updated by adding more information while they are still running. Essentially, a pipeline consists of the following events:
+
+1. The initial running pipeline event with the `status` set to `running`.
+2. Optionally, `N` running pipelines with the same `unique_id` and the `status` set to `running` to update the pipeline with more information.
```

Comment:
```suggestion
2. Optionally, `N` running pipelines with the same `unique_id` and the `status` set to `running`.
```

Reply:
I'm basing this suggestion on the assumption that this isn't directing the user to update the pipeline with more information, but rather describing a potential scenario with multiple running pipelines that have the same `unique_id`. Please let me know if this is inaccurate.

Reply:
The user will be the one updating the pipeline in this case. The user needs to call the API in the way it is described in this section (they will be doing POST requests). The reader can go to the API endpoint for more details but the main goal of this section to describe running pipelines and letting them know, that, optionally, they can update them if they need to.

Reply:
Let me know if something is not clear!

Reply:
Thanks again for clarifying. It sounds like `pipeline event` may be more accurate than `pipeline` in this case. I updated my suggestion accordingly.